templates/assets/js/toggleSidebar.js

```diff
@@ -12,8 +12,16 @@ function toggle() {
 
 $(window).resize(() => {
   const w = $(this).width();
-  if (w < 576) {
+  if (w < 992) {
     document.querySelector(".sidebar").classList.remove("visible");
     document.querySelector(".sidebar").classList.add("hidden");
   }
 });
+
+$(document).ready(() => {
+  const w = $(this).width();
+  if (w > 992) {
```

Comment:
Bootstrap already has a breakpoint defined at >= 992 px. Please see if you can use that by the name for better consistency.
If that is not possible, please take this 992 outside into a variable and name it accordingly maybe breakpoint-lg.
Also you have not covered the case for w=992 here. Please cover it under the first condition.

Reply:
oh yes, sure. I forgot to handle == case. 
Will do these changes.